test/styleToFilter.test.js

```diff
@@ -128,14 +128,32 @@ test('simple style layers', function(t) {
   t.deepEqual(styleToFilter({
     layers: [{
         'source-layer': 'water',
-        filter: ['!=', 'color', 'blue']
+        filter: [
+          'all',
+          [
+              "case",
+              [">=", ["distance-from-center"], 1],
+              true,
+              [">=", ["pitch"], 45]
+          ],
```

Comment:
If we change the expression a little to be:
```
[
              "case",
              [">=", ["distance-from-center"], 1],
              false,
              [">=", ["pitch"], 45]
          ],
```

The case expression will return false, excluding the feature. Therefore, we probably need to aggressively set the entire case expression to `['literal', true]`.